cmd/sidecars/network-passt-binding/BUILD.bazel

```diff
@@ -37,6 +37,7 @@ container_image(
     name = "network-passt-binding-image",
     architecture = select({
         "@io_bazel_rules_go//go/platform:linux_arm64": "arm64",
+        "@io_bazel_rules_go//go/platform:linux_s390x": "s390x",
```

Comment:
Yes. Same for the other features - I'm not aware of s390x lacking any of them. We might find that the architecture needs some specific accommodations, but generally we should aim for feature parity with x86_64, especially since aarch64 is basically there already.

Reply:
I would also prefer feature parity, but i know for some Images already that they do not have the needed s390x base image. Especially cirros and fedora-with-test-tooling.
For the rest, i'm planning on enabling the tests and finding out that way. Since otherwise kubevirt has a lot of images as an output of the build.